lib/libc/musl/src/math/i386/scalbnf.s

```diff
@@ -1,8 +1,3 @@
-.global ldexpf
```

Comment:
In general, when patching libc files like this, a comment like this should be added:

```
/* zig patch: remove ldexpf symbol which is provided by compiler-rt */
```

However, in this particular case, it seems like `scalbnf` is literally equivalent to `ldexpf` (and so on for the others) because `FLT_RADIX == 2`. So I would suggest also adding the `scalbn*` exports to compiler-rt and remove these files entirely.